scripts/aae-export.py

```diff
@@ -1,175 +1,510 @@
+
+        row = layout.row()
+        row.scale_y = 2
+        row.enabled = selected_plane_tracks == 1
+        row.operator("movieclip.aae_export_copy_plane_track")
+
+    @classmethod
+    def poll(cls, context):
+        return context.edit_movieclip is not None
+
+class AAEExportAllTracks(bpy.types.Panel):
+    bl_label = "All tracks"
+    bl_idname = "SOLVE_PT_aae_export_all"
+    bl_space_type = "CLIP_EDITOR"
+    bl_region_type = "TOOLS"
+    bl_category = "Solve"
+    bl_parent_id = "SOLVE_PT_aae_export"
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+        layout.use_property_decorate = False
+        
+        settings = context.screen.AAEExportSettings
+        
+        column = layout.column()
+        column.label(text="Plane tracks")
+        column.prop(settings, "do_includes_power_pin")
+        
+        column = layout.column()
+        column.label(text="All tracks")
+        column.prop(settings, "do_also_export")
+        column.prop(settings, "do_do_not_overwrite")
+        
+        row = layout.row()
+        row.scale_y = 2
+        row.enabled = len(context.edit_movieclip.tracking.tracks) >= 1 or \
+                      len(context.edit_movieclip.tracking.plane_tracks) >= 1
+        row.operator("movieclip.aae_export_export_all")
+
+    @classmethod
+    def poll(cls, context):
+        return context.edit_movieclip is not None
+
+class AAEExportLegacy(bpy.types.Operator, bpy_extras.io_utils.ExportHelper):
     """Export motion tracking markers to Adobe After Effects 6.0 compatible files"""
-    bl_idname = "export.afxkey"
     bl_label = "Export to Adobe After Effects 6.0 Keyframe Data"
+    bl_idname = "export.aae_export_legacy"
     filename_ext = ""
-    filter_glob = StringProperty(default="*", options={'HIDDEN'})
+    filter_glob = bpy.props.StringProperty(default="*", options={"HIDDEN"})
 
     def execute(self, context):
-        return write_files(self.filepath, context)
-
-classes = (
-    ExportAFXKey,
-)
-
-def menu_func_export(self, context):
-    self.layout.operator(ExportAFXKey.bl_idname, text="Adobe After Effects 6.0 Keyframe Data")
-
+        # This is broken but I don't want to fix...
+        if len(bpy.data.movieclips) != 1:
+            raise ValueError("The legacy export method only allows one clip to be loaded into Blender at a time. You can either try the new export interface at „Clip Editor > Tools > Solve > AAE Export“ or use „File > New“ to create a new Blender file.")
+        clip = bpy.data.movieclips[0]
+        settings = context.screen.AAEExportSettings
+
+        for track in clip.tracking.tracks:
+            AAEExportExportAll._export_to_file(clip, track, AAEExportExportAll._generate(clip, track, False), self.filepath, True)
+
+        for plane_track in clip.tracking.plane_tracks:
+            AAEExportExportAll._export_to_file(clip, track, AAEExportExportAll._generate(clip, plane_track, False), self.filepath, True)
+
```

Comment:
The function should return a result here, probably just `{"FINISHED"}`. Otherwise the export runs through but still shows an error message at the end.